pkgs/misc/vim-plugins/default.nix

```diff
@@ -2,7 +2,7 @@
 { fetchurl, bash, stdenv, python, go, cmake, vim, vimUtils, perl, ruby, unzip
 , which, fetchgit, fetchFromGitHub, fetchhg, fetchzip, llvmPackages, zip
 , vim_configurable, vimPlugins, xkb_switch, git
-, Cocoa
+, Cocoa ? null
```

Comment:
It causes build failures on non-darwin, at least for me. It needs handling in the outer scope.